### ISSUE TITLE
Can entity references and family define the structure of GFM?

### ISSUE THREAD

**Issue by johelegp** [github/cmark-gfm#147]
https://github.com/github/cmark-gfm/blob/87c0139ee1213528f4f24e08e6844d261b8d0f1b/test/spec.txt#L5938-L5940

This doesn't include the GFM extensions. I guess it'd be better if the upstream spec said "defined in this document" rather than "in CommonMark".

**Comment by kivikakk** [comment]
Indeed, the spec references CommonMark many times, where strictly interpreting it that way would cause confusion. ("A sequence of characters is a valid CommonMark document." "The contents of the heading are the result of parsing the preceding lines of text as CommonMark inline content." "In CommonMark, a list can interrupt a paragraph.") I think it should be fairly obvious that such an interpretation is not intended or useful.
